http-proxy/Cargo.toml

```
@@ -1,21 +1,15 @@
 [package]
-name = "http-proxy"
+name = "http-full"
```

Ah, the name is still invalid.
```suggestion
name = "http-proxy"
```

---

Oops, I feel stupid now. Fixed!